arcgis-ios-sdk-samples/Edit data/Add features with contingent values/AddFeaturesContingentValuesViewController.swift

```diff
@@ -0,0 +1,200 @@
+            featureTable?.load { [weak self] error in
+                guard let self = self else { return }
+                if let error = error {
+                    self.presentAlert(error: error)
+                } else {
+                    // Create and load the feature layer from the feature table.
+                    let featureLayer = AGSFeatureLayer(featureTable: self.featureTable!)
+                    // Add the feature layer to the map.
+                    self.mapView.map?.operationalLayers.add(featureLayer)
+                    // Set the map's viewpoint to the feature layer's full extent.
+                    let extent = featureLayer.fullExtent
+                    self.mapView.setViewpoint(AGSViewpoint(targetExtent: extent!))
+                    // Add buffer graphics for the feature layer.
+                    self.queryFeatures()
+                }
+            }
+        case .failure(let error):
+            presentAlert(error: error)
+        }
+    }
+    
+    /// Creates a map with a vector tiled layer basemap.
+    func makeMap() -> AGSMap {
+        // Get the vector tiled layer by name.
+        let fillmoreVectorTiledLayer = AGSArcGISVectorTiledLayer(name: "FillmoreTopographicMap")
+        // Use the vector tiled layer as a basemap.
+        let basemap = AGSBasemap(baseLayer: fillmoreVectorTiledLayer)
+        let map = AGSMap(basemap: basemap)
+        return map
+    }
+    
+    /// Create buffer graphics for the features.
+    func queryFeatures() {
+        // Create the query parameters.
+        let queryParameters = AGSQueryParameters()
+        // Set the where clause to filter for buffer sizes greater than 0.
+        queryParameters.whereClause = "BufferSize > 0"
+        // Create an array of graphics to add to the graphics overlay.
+        var graphics = [AGSGraphic]()
+        // Query the features with the query parameters.
+        featureTable?.queryFeatures(with: queryParameters) { [weak self ] result, error in
+            guard let self = self else { return }
+            if let error = error {
+                // Present an alert if there is an error while querying the features.
+                self.presentAlert(error: error)
+            } else if let result = result {
+                // Get the array of features from the query result.
+                let features = result.featureEnumerator().allObjects
+                // For each feature, add the buffer symbol according to its buffer size.
+                features.forEach { feature in
+                    let graphic = self.createGraphics(for: feature)
+                    graphics.append(graphic)
+                }
+                // Add the graphics to the graphics overlay.
+                self.graphicsOverlay.graphics.addObjects(from: graphics)
+                // Add the graphics overlay to the map view.
+                self.mapView.graphicsOverlays.add(self.graphicsOverlay)
```

Comment:
Although the `queryFeatures` method is only called once and there is no side effect to add graphics overlay to the map view here, I still suggest adding the graphics overlay in mapView didSet, as well as to clear its graphics before adding new ones.